samples/v1.0/HostConfig/sampleFontStyles.json

```diff
@@ -0,0 +1,155 @@
+{
+	"spacing": {
+		"small": 3,
+		"default": 8,
+		"medium": 20,
+		"large": 30,
+		"extraLarge": 40,
```

Comment:
The host config folder has been moved out of the v1.0 folder, so you'll need to move your new files to the new location.